pimd/pim_cmd.c

```diff
@@ -5623,8 +5621,8 @@ DEFUN (show_ip_pim_nexthop_lookup,
 	}
 
 	pim_addr_dump("<grp?>", &grp, grp_str, sizeof(grp_str));
-	pim_addr_dump("<nexthop?>", &nexthop.mrib_nexthop_addr,
-		      nexthop_addr_str, sizeof(nexthop_addr_str));
+	snprintfrr(nexthop_addr_str, sizeof(nexthop_addr_str), "%pPAs",
```

Comment:
Do we need this statement? Cant we directly use %pPA in the vty_out below.

Reply:
fixed